parts/linux/cloud-init/artifacts/components.json

```diff
@@ -421,6 +421,13 @@
               "1.1.9-5.cm2"
             ]
           }
+        },
+        "azurelinux": {
+          "current": {
+            "versions": [
+              "1.1.12-1.azl3"
```

Comment:
did we run a test build to make sure these are getting picked up and installed properly?

Reply:
Yeah, I have been running the test vhd pipeline to make sure things look good.

Here is a pipeline link for it - https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=101455732&view=logs&j=fa0bd643-15ef-5339-b190-0ae2e9563c80&t=45c1fbf4-8441-5ec1-7770-87c1e2689273&l=4587